src/app/components/flyout/index.html

```diff
@@ -39,52 +34,54 @@
       propertyName="ariaDescribedBy"
       isOptional="true"
     >
-      Sets the flyout's <stache-code>aria-describedby</stache-code> attribute to support accessibility. The value should be the HTML element ID (without the leading <stache-code>#</stache-code>) of the element that describes the flyout. Typically, this is the text on the flyout but does not include items that users interact with such as buttons and forms.
+      Sets the flyout's <stache-code>aria-describedby</stache-code> attribute to support accessibility. The value should be the HTML element ID (without the leading <stache-code>#</stache-code>) of the element that describes the flyout. Typically, the description includes text on the flyout but not on items that users interact with such as buttons and forms. This property accepts <stache-code>string</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="ariaLabelledBy"
       isOptional="true"
     >
-      Sets the flyout's <stache-code>aria-labelledby</stache-code> attribute to support accessibility. The value should be the HTML element ID (without the leading <stache-code>#</stache-code>) of the element that labels the flyout.
+      Sets the flyout's <stache-code>aria-labelledby</stache-code> attribute to support accessibility. The value should be the HTML element ID (without the leading <stache-code>#</stache-code>) of the element that labels the flyout. This property accepts <stache-code>string</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="ariaRole"
       isOptional="true"
     >
-      Sets the flyout's <stache-code>role</stache-code> attribute to support accessibility.
+      Specifies an ARIA role for the flyout to support accessibility by indicating how the flyout functions and what it controls. The ARIA role indicates what the flyout represents on the web page. For information about ARIA roles, see the <a href="https://www.w3.org/WAI/PF/aria/roles">WAI-ARIA roles model</a>. This property accepts <stache-code>string</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="defaultWidth"
       defaultValue="500px"
```

Comment:
Now that we're pointing out this is a `numeric` type, we should probably change `defaultValue="500px"` to `defaultValue="500"`

Reply:
Will do.